impexp-client/src/main/java/org/citydb/modules/kml/gui/preferences/GeneralPanel.java

```diff
@@ -317,6 +327,19 @@ public void actionPerformed(ActionEvent e) {
 				browseGltfConverterFile(Language.I18N.getString("pref.kmlexport.dialog.gltf.title"));
 			}
 		});
+
+		// when glb is enabled, always embed textures
+		exportGltfBinary.addItemListener(new ItemListener() {
+			@Override
+			public void itemStateChanged(ItemEvent e) {
+				if (exportGltfBinary.isSelected()) {
+					embedTexturesInGltfCheckbox.setEnabled(false);
+					embedTexturesInGltfCheckbox.setSelected(true);
+				} else {
+					embedTexturesInGltfCheckbox.setEnabled(true);
+				}
+			}
+		});
```

Comment:
Was NOT moved to branch [glb-draco-support](https://github.com/3dcitydb/importer-exporter/tree/glb-draco-support).